packages/styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livekit/components-styles",
-  "version": "1.1.4",
+  "version": "1.1.5",
```

Comment:
versions are automatically bumped by the CI, bumping them in PRs has the effect of skipping versions or unintentionally publishing them